bin/i18n/sync-codeorg-all

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env ruby
```

Comment:
Presumably this is meant to only be run on staging? A file comment indicating what the script does and this restriction seems worthwhile.